Guide/database.markdown

```diff
@@ -567,6 +567,8 @@ do
     deleteRecord user
 ```
 
+Or pass only the ID with `[`deleteRecordById`](https://ihp.digitallyinduced.com/api-docs/IHP-ModelSupport.html#v:deleteRecordById).
```

Comment:
I'd actually put this under it's own headline `Deleting a single record by id`. Then it's easy to find when using the navigation at the top of the page :)